templates/shield-study/.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             - firefox-beta
             - firefox-release
             - firefox-unbranded-release
-          key: dependencies-and-browsers-{{ checksum "package.json" }}-20180831
+          key: dependencies-and-browsers-\{\{ checksum "package.json" }}-20180831
```

Comment:
These are wrong and are bugs, but I'm a bit lost as to how to solve it...

Currently **plop** thinks that the .circleci/config.yml file is using handlebars variables and that "checksum" is a helper function. I haven't found a way to tell **plop**/`addMany` to just copy the file and NOT do variable interpolation, so I ended up adding backslashes here so the study gets created, but it unfortunately breaks the CircleCI config file since those backslashes get copied into the final template output files.

```sh
$ npx pdehaan/create-shield-study#general-fixes ruh-roh

? Shield (or Pioneer) shield
? inform Strategy and Insights about your project Yes
? choose all starting libraries shield-study-addon-utils
[FAILED] addMany Missing helper: "checksum"
[FAILED] info Aborted due to previous action failure
```

Reply:
Filed upstream since this feels funny (and I forgot to do it last week): https://github.com/amwmedia/plop/issues/144